monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -2872,6 +2872,12 @@ object Task extends TaskInstancesLevel1 {
   /** Builds a [[Task]] of Right */
   def right[A, B](a: B): Task[Either[A, B]] = Task.pure(Right(a))
 
+  /** A [[Task]] of None */
+  def none[A]: Task[Option[A]] = Task.pure(Option.empty[A])
```

Comment:
maybe `Now(Option.empty[A])`? 
for consistency with the rest of the code here and less calls